packages/p2p-media-loader-core/src/segments-storage.ts

```diff
@@ -67,7 +69,7 @@ export class SegmentsMemoryStorage {
       lastAccessed: performance.now(),
     });
     this.logger(`add segment: ${id}`);
-    this.dispatchStorageUpdatedEvent(segment.stream);
+    this.onStorageUpdated(segment.stream);
```

Comment:
This is different from the original code, which dispatches not `onStorageUpdated` here but `onStorageUpdated${StreamUtils.getStreamShortId(stream)}` event